app/domain/operations/fdsh/ridp/publish_primary_request.rb

```diff
@@ -22,7 +22,7 @@ def call(payload)
 
         def build_event(payload)
           hbx_id = payload.to_h[:family_members].detect{|fm| fm[:is_primary_applicant] == true}[:person][:hbx_id]
-          event('events.fdsh.ridp.primary_determination_requested', attributes: payload.to_h, headers: { correlation_id: hbx_id })
+          event('events.fdsh.ridp.primary_determination_requested', attributes: payload.to_h, headers: { correlation_id: hbx_id, payload_format: EnrollRegistry[:ridp_h139].setting(:payload_format).item })
```

Comment:
any particular reason we didn't assign this to a variable like the `hbx_id`? 

Either way is fine with me, it was just the difference of the two that made me pause. Admittedly the assignment on line 24 is slightly more complex than the line that pulls the payload format from the settings, so maybe this is the most sensible solution?

Reply:
It s directly taking the setting from the RR, it is not being re-used, so went with the simplest way forward